android/assets/jsons/translations/Lithuanian.properties

```diff
@@ -2149,5701 +1424,3244 @@ ConditionalsPlacement =
+[relativeAmount]% growth [cityFilter] = [relativeAmount]% prie augimo: [cityFilter]
+Monarchy = Monarchija
+Tradition Complete = Tradicija +
+Tradition = Tradicija
+
+Collective Rule = Kolektyvinis valdymas
+Citizenship = Pilietybė
+Republic = Respublika
+Representation = Atstovavimas
+Each city founded increases culture cost of policies [relativeAmount]% less than normal = Kiekvienas įkurtas miestas [relativeAmount]% mažiau didina kultūros kainą santvarkos institutams
+Meritocracy = Meritokratija
+Liberty Complete = Išlaisvėjimas +
+Free Great Person = Didis Žmogus dovanai
+Liberty = Išlaisvėjimas
+
+Warrior Code = Priesaika
+Discipline = Disciplina
+when adjacent to a [mapUnitFilter] unit = kada šialia yra: [mapUnitFilter]
+Military Tradition = Karinė tradicija
+[relativeAmount]% XP gained from combat = [relativeAmount]% patirties kariaujant
+Military Caste = Karių kasta
+Professional Army = Profesionali kariuomenė
+Honor Complete = Garbė +
+Honor = Garbė
+vs [mapUnitFilter] units = prieš [mapUnitFilter] dalinius
+Notified of new Barbarian encampments = Pranešti apie naujas barbarų stovyklas
+
+Organized Religion = Organizuota religija
+Mandate Of Heaven = Dievo Įsakymai
+[relativeAmount]% of excess happiness converted to [stat] = [relativeAmount]% nuo perteklinės laimės paverčiama į: [stat]
+Theocracy = Teokratija
+[relativeAmount]% [stat] from every [tileFilter/specialist/buildingName] = [relativeAmount]% [stat] nuo kiekvieno: [tileFilter/specialist/buildingName]
+Reformation = Reformacija
+Free Religion = Religija dovanai
+Piety Complete = Dievotumas +
+Piety = Dievotumas
+before adopting [policy] = prieš priimant [policy]
+
+Philantropy = Filantropija
+Gifts of Gold to City-States generate [relativeAmount]% more Influence = Laisviesiems miestams dovanojamas auksas duoda [relativeAmount]% daugiau įtakos
+Aesthetics = Estetika
+Resting point for Influence with City-States is increased by [amount] = Bazinė įtaka laisviesiems miestams padidinama: [amount]
+Scholasticism = Skolastika
+Allied City-States provide [stat] equal to [relativeAmount]% of what they produce for themselves = Laisvieji miestai-sąjungininkai duoda [relativeAmount]% nuo jų gaminamo(s): [stat]
+Cultural Diplomacy = Kultūrinė Diplomatija
+[relativeAmount]% resources gifted by City-States = [relativeAmount]% prie resursų, jum dovanojamų iš laisvųjų miestų
+[relativeAmount]% Happiness from luxury resources gifted by City-States = [relativeAmount]% prie laimės nuo retų resursų, gaunamų iš laisvųjų miestų
+Educated Elite = Išsilavinusi Elita
+Allied City-States will occasionally gift Great People = Laisvieji miestai-sąjungininkai kartais dovanos jums Didžiuosius žmones
+Patronage Complete = Patronažas +
+Influence of all other civilizations with all city-states degrades [relativeAmount]% faster = Kitų civilizacijų įtaka visiems laisviesiems miestams mažėja [relativeAmount]% greičiau
+Triggers the following global alert: [param] = Siunčia globalų pranešimą: [param]
```

Comment:
And this the second, though it looks like it's not your mistake after all.

Reply:
Ah, no, you *do* need to leave the generated parameter name in place. Both need to be `[comment]`. I thought both named equal would suffice, but our UniqueType system replaces the parameter name with the "supposed to" name before the test. In-game your line would actually work, but our tests demand you leave the left side before the " = " alone...

Reply:
changed to "comment"
... in 4.0.11 there was "param"
i thought that checker simply will remove such line and add new one with "requires translation"

in my translation there are some lines witch not exists only in mods,
also lines whose do not exists even in any mods
imho checker must just remove those lines without processing them

Reply:
> in 4.0.11-4.0.15 there was "param"

Exactly!

> lines whose do not exists even in any mods

Two versions: The line comes from templates.properties -> technically entirely possible something is left unused and forgotten after a code change. Point them out - separate issue - or hijack that issue dealing with missing translation entries. V2 - not from templates, then must come from the json scan. If _that_ gives unused lines that would be even more curious - and report-worthy. For a translation PR, just ignore and move to more important lines if you want.